aspnetcore/fundamentals/middleware/extensibility-third-party-container/sample/Middleware/SimpleInjectorMiddlewareFactory.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Microsoft.AspNetCore.Http;
+using SimpleInjector;
+
+namespace MiddlewareExtensibilitySample.Middleware
+{
+    #region snippet1
+    public class SimpleInjectorMiddlewareFactory : IMiddlewareFactory
+    {
+        private readonly Container _container;
+
+        public SimpleInjectorMiddlewareFactory(Container container)
+        {
+            _container = container;
+        }
+
+        public IMiddleware Created { get; private set; }
```

Comment:
what are these properties for?

Reply:
I'll remove them ... just copying your default middleware factory.

Stand-by tho ... don't look too close yet ... I have a few more updates here.

Reply:
Release should be handled by the container